src/Cassandra/Metrics/AppMetricsImpl/CassandraDriverAppMetricsReportingExtensions.cs

```diff
@@ -0,0 +1,25 @@
+#if NETSTANDARD2_0
+using System;
+using App.Metrics;
+using App.Metrics.Builder;
+using App.Metrics.Formatters.Graphite;
+using App.Metrics.Reporting.Graphite;
+
+namespace Cassandra.Metrics.AppMetricsImpl
+{
+    public static class CassandraDriverAppMetricsReportingExtensions
+    {
+        public static IMetricsBuilder ToGraphiteForDriver(this IMetricsReportingBuilder reportingBuilder, Uri graphiteUri)
+        {
+            return reportingBuilder.ToGraphite(new MetricsReportingGraphiteOptions
+            {
+                Graphite = new GraphiteOptions(graphiteUri),
+                MetricsOutputFormatter = new MetricsGraphitePlainTextProtocolOutputFormatter(new MetricsGraphitePlainTextProtocolOptions
+                {
+                    MetricPointTextWriter = new CassandraDriverGraphitePointTextWriter(false, "env", "app", "server")
+                })
```

Comment:
I implement `CassandraDriverGraphitePointTextWriter` because the behavior of `App.Metrics` is awkward for me: it prepends the metric type at the beginning of the path (so you have `counter.Environment.Host.Application.MetricPath` which is an uncomfortable way for organizing metrics). 
Maybe we need to remove this wrapper from core driver library